### DISCUSSION TITLE
Camera Path Export Feature

### DISCUSSION THREAD

**Discussion by ClockTheSenior** [Sam54123/mc-world-export#57] [Ideas]
It would be great if you could export the existing camera path of the replay, directly with the mod/addon into blender! You could add a feature in the import options to import the replay mod camera movements!
I know you can do it manually too, but sometimes you have to fiddle around with dimensions and other stuff + you could add a feature for the effects and the FOV! The FOV could match the game/replay camera FOV and the effects could also be emulated by adding a camera overlay. The effects would only really make sense in the player perspective thingy. lol

**Comment by Igrium** [comment]
This is a planned feature! At least the camera animation is. As for effects, the implementation of them will vary greatly depending on your style and workflow, so that's best left for the artist to figure out.
